custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -69,31 +69,33 @@ def aggregation_query(self):
             1,
             'no',
             5,
-            CASE WHEN
-                (count(*) filter (WHERE date_trunc('MONTH', vhsnd_date_past_month) = %(start_date)s))>0
-                THEN 1 ELSE 0 END,
-            CASE WHEN
-                (count(*) filter (WHERE date_trunc('MONTH', date_cbe_organise) = %(start_date)s))> 1
-                THEN 1 ELSE 0 END,
+            CASE WHEN vhnd_conducted is not null and vhnd_conducted>0 THEN 1 ELSE 0 END,
+            CASE WHEN cbe_conducted is not null and cbe_conducted>1 THEN 1 ELSE 0 END,
```

Comment:
There was a message on the skype dashboard chat:

> CBEs and VHSNDs are killed 
> Those indicators are no longer required, basically

is this still needed?

Reply:
yeah. Those conversations were for something else not for our dashboard